spec/features/users/dashboard/experienced_user_spec.rb

```diff
@@ -84,7 +103,14 @@
 
             within "#gym-#{gym.yelp_gym_id}" do
               expect(page).to have_content(gym.name)
-              # expect(page).to have_content(gym.adress)
+              # TODO: Once we have the GymMembership endpoint returning the gym
+              #       address, we can add the address attributes to the poro and
+              #       uncomment this line below.  See the Event poro for how the
+              #       city_state_zip can get parsed.
+              # expect(page).to have_content(gym.adress1)
+              # expect(page).to have_content(gym.adress2)
+              # expect(page).to have_content(gym.adress3)
+              # expect(page).to have_content(gym.city_state_zip)
```

Comment:
Note to team - BE update in progress via this [issue](https://github.com/tvaroglu/spot_me_backend/issues/62)